source/collimation.f90

```diff
@@ -1821,11 +1822,13 @@ subroutine collimate_start
       call f_requestUnit('all_absorptions.dat', all_absorptions_unit)
       call f_requestUnit('Coll_Scatter.dat', coll_scatter_unit)
       call f_requestUnit('FirstImpacts.dat', FirstImpacts_unit)
+      call f_requestUnit('JawProfiles.dat', profiling_jaw_unit)
```

Comment:
Please add the filenames as a constant in the module header. I think my updates on this are in the branch not yet merged. Also, we should avoid using mix of upper and lower case characters in new file names as different operating systems have different rules for case sensitivity.